client/src/pages/CreateUser.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useState } from "react";
+import Parse from "../env.Backend/env.parseConfig.ts";
+import "../App.css";
+import Layout from "./Layout.tsx";
+
+function CreateUser() {
+  const [userData, setUserData] = useState({
+    username: "",
+    firstName: "",
+    lastName: "",
+    email: "",
+    password: "",
+    ProfilePicture: "", // Stored as base64
+    BirthDate: "", // User's birth date
+  });
```

Comment:
As we discussed in the exercise class, please consider modifying how your backend manages user creation and authenticating existing users by following these guidelines:
     - https://www.back4app.com/docs/react/working-with-users/sign-up-page-react
     - https://www.back4app.com/docs/react/working-with-users/react-login
     - https://www.back4app.com/docs/react/working-with-users/get-current-user-react
     
After modifying the above, please take a look at the guide below to update your data relationships if necessary: 
     - https://www.back4app.com/docs/react/data-objects/relationships